src/main/kotlin/no/fdk/fdk_reasoning_service/cache/ReferenceDataCache.kt

```diff
@@ -61,7 +61,7 @@ class ReferenceDataCache(private val uris: ApplicationURI) {
         invalidateAndUpdateFileTypes()
     }
 
-    @Scheduled(cron = "10 */3 * * * ?")
+    @Scheduled(cron = "0 10 */3 * * ?")
```

Comment:
Er det riktig at kolonne nr 2 bestemmer antall timer interval?

`<minute> <hour> <day-of-month> <month> <day-of-week> <command>`

https://www.baeldung.com/cron-expressions

Reply:
Scheduled i spring bruker sekund i første kolonne av cron: https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/scheduling/annotation/Scheduled.html#cron()

Reply:
Den var ihvertfall ikke riktig slik den er nå, for den kjørte jobb hvert 3. minutt :sweat_smile: